src_Core/RegFiles/CSR_RegFile_MSU.bsv

```diff
@@ -339,6 +348,16 @@ deriving (Eq, Bits, FShow);
 
 // ================================================================
 
+`ifdef PERFORMANCE_MONITORING
+(* synthesize *)
+module mkPerfCountersFlute (PerfCounters_IFC #(No_Of_Ctrs, Counter_Width, No_Of_Evts));
```

Comment:
This file is also used by Piccolo so this is not a good name. Why not just `mkPerfCountersSynth` or a similarly generic name?